src/aleph/netty.clj

```diff
@@ -1604,7 +1722,9 @@ initialize an DnsAddressResolverGroup instance.
                    (d/finally'
                      ;; 3. At this stage, stop the EventLoopGroup, this will cancel any
                      ;;    in flight pending requests.
-                     #(.shutdownGracefully group 0 0 TimeUnit/SECONDS)))
+                     ;;    We still wait 1.5s to gracefully end repeating tasks
+                     ;;    like the date-header-value.
+                     #(.shutdownGracefully group 1500 1500 TimeUnit/MILLISECONDS)))
```

Comment:
I only noticed this now when working on the follow-up change for the ALPN config: can you explain the rationale behind this change a bit more? I was confused where that 1.5s slowdown on HTTP tests was coming from and only then found this. Why is it important to keep the date-header-value task running when the server is already shutdown? And if it's not essential, I would tend to introduce another setting for overriding this - our test suite contains hundreds of tests which start and stop aleph servers so 1.5 seconds on each would add up quite badly :grimacing:

Reply:
The issue was some race condition causing `test-request-timeout` to fail. 

I agree that there's no need to delay shutdown just for one test. I think I was really exasperated at the time I made the change. I'll see about undoing it.

Reply:
Ehh, whatever situtation led me to change that no longer holds. I can't replicate the error now.

Reply:
Ah nice, glad to hear :100: